docs/Agent/user-guide/man_page.md

```diff
@@ -315,17 +324,29 @@ Show this message and exit.
 `pbench-register-tool-set` - register the specified toolset
 
 **SYNOPSIS**  
-`pbench-register-tool-set [--group=<group-name>] [--interval=<interval>] [--no-install] [--remotes=<remote-host>[,<remote-host>]] [--labels=<label>[,<label>]] [--help]`
-
-`pbench-register-tool-set [--group=<group-name>] [--interval=<interval>] [--no-install] [--remotes=@<remotes-file>] [--help]`
+`pbench-register-tool-set [OPTIONS]`
```

Comment:
Since the tool set is a required argument, I would include it in the command synopsis:
```
**SYNOPSIS**  
`pbench-register-tool-set [OPTIONS] <tool-set>`
```
(This will make the references at lines 332 and 341 make more sense.)